graylog2-web-interface/src/util/StringUtils.ts

```diff
@@ -50,6 +50,34 @@ const StringUtils = {
 
     return text;
   },
+  getRecursiveChildText(reactNode: string|React.ReactNode) {
+    if (typeof reactNode === 'string') {
+      return reactNode;
+    }
+
+    const { children } = (reactNode as any)?.props || {};
+
+    if (Array.isArray(reactNode)) {
+      const joinedNodes = [];
+
+      reactNode.forEach((node) => {
```

Comment:
Why not use `map` instead?

Reply:
Because the output array isn't guaranteed to have the same length as the input array.